app/controllers/users_controller.rb

```diff
@@ -10,23 +10,24 @@ def new
     @user = User.new(user_params)
   end
 
-  def create
-    binding.pry
+  def auth
```

Comment:
So I was doing something different with this.  I was thinking we do a User.find_by :email. Then if the User exists it will create the session and redirect to their dashboard, much like you have.  I just don't have the create built in there because it will instead reroute to the registration page since we need additional information to create a user account.